WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/ApplicationLogViewController.swift

```diff
@@ -212,20 +212,14 @@ private extension ApplicationLogViewController {
     func logFileWasPressed(in row: Int) {
         let logFileInfo = logFiles[row]
 
-        let identifier = ApplicationLogDetailViewController.classNameWithoutNamespaces
-        guard let appLogDetailVC = UIStoryboard.dashboard.instantiateViewController(identifier: identifier) as? ApplicationLogDetailViewController else {
-            DDLogError("Error: attempted to instantiate ApplicationLogDetailViewController. Instantiation failed.")
-            return
-        }
         do {
             let contents = try String(contentsOfFile: logFileInfo.filePath)
             let date = dateFormatter.string(from: logFileInfo.creationDate )
-            appLogDetailVC.logText = contents
-            appLogDetailVC.logDate = date
+            let appLogDetailVC = ApplicationLogDetailViewController(with: contents, for: date)
+            navigationController?.pushViewController(appLogDetailVC, animated: true)
```

Comment:
nit: show()
```suggestion
            show(appLogDetailVC, sender: self)
```

Reply:
🤯 how did I not know about `show()`?! The documentation says it's been around since iOS 8. I'm so very confused 🙃